tests/integration/test_batch.py

```diff
@@ -1,20 +1,26 @@
 import time
-
-from labelbox.exceptions import ProcessingWaitTimeout
 import pytest
+
+from uuid import uuid4
 from labelbox import Dataset, Project
+from labelbox.exceptions import ProcessingWaitTimeout, MalformedQueryException, ResourceConflict, LabelboxError
 
 IMAGE_URL = "https://storage.googleapis.com/diagnostics-demo-data/coco/COCO_train2014_000000000034.jpg"
+EXTERNAL_ID = "my-image"
+
+
+def get_data_row_ids(ds: Dataset):
+    return [dr.uid for dr in list(ds.export_data_rows())]
```

Comment:
export_data_rows() is legacy exports. Could you use exports_v2 here?

Reply:
well we are still need to support exports v1 .. it is not retired yet

Reply:
i think it is good to cover legacy functions as long as they are being part of the SDK.